FAQ.md

```diff
@@ -189,16 +187,15 @@ Due to a bug in Android this is not possible.
 See [here](https://github.com/M66B/NetGuard/issues/42) for more information.
 
 <a name="FAQ24"></a>
-**(24) Can you remove the notification from the status bar?**
+**(24) Can you remove the notification from the notification screen?**
```

Comment:
Oh, ok, for now I only saw that behaviour: **high w/ icon** or **low w/o icon**.